dencam.py

```diff
@@ -53,9 +53,11 @@ def cleanup(flags):
         recorder = Recorder(configs)
         number_of_states = len(STATE_LIST)
         state = State(number_of_states)
+        airplane_mode = AirplaneMode(configs)
```

Comment:
I am still wondering whether it would be better to create the airplane_mode object solely within ButtonHandler (as ButtonHandler is its only client) and just give configs as an argument to the ButtonHandler constructor.  Did you determine a reason that you wanted to stick with creating the AirplaneMode object in dencam.py?